jgrapht-core/src/test/java/org/jgrapht/GraphTestsTest.java

```diff
@@ -334,11 +334,10 @@ public void testIsForest1()
     @Test
     public void testIsForest2()
     {
-        Graph<Integer, DefaultEdge> g = new Pseudograph<>(DefaultEdge.class);
+        Graph<Integer, DefaultEdge> g = new Pseudograph<>(SupplierUtil.createIntegerSupplier(), SupplierUtil.DEFAULT_EDGE_SUPPLIER, false);
```

Comment:
This is quite long, especially since we use this in virtually every test. We might need to create some factory method in the Test suite, or add a default constructor as shorthand.